src/main/java/org/truffleruby/platform/DefaultRubiniusConfiguration.java

```diff
@@ -57,27 +56,6 @@
     private static final int SIZE_OF_POINTER = 8;
 
     public static void load(RubiniusConfiguration configuration, RubyContext context) {
-        configuration.config("rbx.platform.file.S_IRUSR", FileStat.S_IRUSR);
-        configuration.config("rbx.platform.file.S_IWUSR", FileStat.S_IWUSR);
-        configuration.config("rbx.platform.file.S_IXUSR", FileStat.S_IXUSR);
-        configuration.config("rbx.platform.file.S_IRGRP", FileStat.S_IRGRP);
-        configuration.config("rbx.platform.file.S_IWGRP", FileStat.S_IWGRP);
-        configuration.config("rbx.platform.file.S_IXGRP", FileStat.S_IXGRP);
-        configuration.config("rbx.platform.file.S_IROTH", FileStat.S_IROTH);
-        configuration.config("rbx.platform.file.S_IWOTH", FileStat.S_IWOTH);
-        configuration.config("rbx.platform.file.S_IXOTH", FileStat.S_IXOTH);
-        configuration.config("rbx.platform.file.S_IFMT", FileStat.S_IFMT);
-        configuration.config("rbx.platform.file.S_IFIFO", FileStat.S_IFIFO);
-        configuration.config("rbx.platform.file.S_IFCHR", FileStat.S_IFCHR);
-        configuration.config("rbx.platform.file.S_IFDIR", FileStat.S_IFDIR);
-        configuration.config("rbx.platform.file.S_IFBLK", FileStat.S_IFBLK);
-        configuration.config("rbx.platform.file.S_IFREG", FileStat.S_IFREG);
-        configuration.config("rbx.platform.file.S_IFLNK", FileStat.S_IFLNK);
-        configuration.config("rbx.platform.file.S_IFSOCK", FileStat.S_IFSOCK);
-        configuration.config("rbx.platform.file.S_ISUID", FileStat.S_ISUID);
-        configuration.config("rbx.platform.file.S_ISGID", FileStat.S_ISGID);
-        configuration.config("rbx.platform.file.S_ISVTX", FileStat.S_ISVTX);
-
         for (Fcntl fcntl : Fcntl.values()) {
```

Comment:
It looks like there are timeval values in the platform-specific files as well. Is that data incomplete? Or should we remove additional code here?

Reply:
> timeval values

What do you mean? Fcntl values?
I think we can clean this up further in another PR, yes.
FileStat is part of jnr-posix so I had to fix this usage.

Reply:
It's lower down in the file, not part of the diff. I think the fnctl ones are a bit different because jnr-constants would have entries for values not defined on a particular platform.

Anyway, you answered my question. We can clear out more in another PR. I just didn't know if you removed some and not others for a technical reason.